lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -16,14 +16,14 @@ class InstructionSlide extends React.Component {
           <div className="wrapper">
 
             <div className="container__row">
-              <div className="container__block">
-                <h2 className="heading">This is how you do it.</h2>
+              <div className="container__block -narrow -centered">
+                <h2 className="heading -gamma">This is how you do it.</h2>
                 <p>We are PUMPED to see your photos. Complete these simple steps and upload a photo of yourself in action. Your pics will inspire others to join the movement &ndash; so don't forget to share them!</p>
               </div>
```

Comment:
@DFurnes I can fix this in another PR & I'll just through all of the slides I guess